analyses/snv-callers/README.md

```diff
@@ -5,7 +5,27 @@ This consensus mutation file is [MAF-like](#consensus-mutation-call) meaning it
 
 The resulting MAF and TMB files from this analysis are saved to `snv-callers/results/consensus/` folder and used by downstream analyses.
 
-See the comparison results plots [here](https://cansavvy.github.io/openpbta-notebook-concept/snv-callers/compare_snv_callers_plots.nb.html).
+Consensus mutations saved to `pbta-snv-consensus-mutation.maf.tsv.gz` are decided from agreement of [3 out of 3 of the snv callers used](#02-merge_callersr).
+Four callers were originally run but VarDict was dropped from downstream analysis because it called a large number (~39 million) of very low VAF mutations that were unique to it, suggesting that most of these were likely to be false positive calls.
+~1.2 million mutations were called by all three of Mutect2, Strelka2, and Lancet and ended up in the consensus file. 
+And between ~73k and ~1 million mutations were shared between combinations of any two these callers.
```

Comment:
I think having the total addtional could be useful here: something like: 
```suggestion
Between ~73k and ~1 million mutations were shared between combinations of any two these callers, for a total of X million additional SNP calls if we had adopted a 2 out of 3 consensus.
```

Reply:
Okay. I'll get that number then.

Reply:
I got this number in a janky way that I don't think is accurate so I need to revaluate where and how I should get this number in a reproducible and nice way.